variables.tf

```diff
@@ -263,108 +263,10 @@ variable "custom_container_definitions" {
   default     = ""
 }
 
-variable "entrypoint" {
```

Comment:
@druchoo it looks like your merge to master didn't go cleanly - these variables shouldn't be removed

Reply:
🤦